modules/crawler/pipe/save_snapshot.go

```diff
@@ -35,6 +37,13 @@ type SaveSnapshotToDBJoint struct {
 
 const compressEnabled ParaKey = "compress_enabled"
 const bucket ParaKey = "bucket"
+const snapshottimeToless ParaKey = "snapshottime_toless"
```

Comment:
for the naming, since there is already in the snapshot joint, remove the prefix: "snapshot" maybe more clean to read, how about `accelerate_steps` , `decelerate_steps`  and `max_revision`